src/sentry/incidents/serializers/alert_rule.py

```diff
@@ -450,7 +450,7 @@ def _handle_triggers(self, alert_rule, triggers):
                         "user": self.context["user"],
                         "use_async_lookup": self.context.get("use_async_lookup"),
                         "input_channel_id": self.context.get("input_channel_id"),
-                        "validate_channel_id": self.context.get("validate_channel_id"),
+                        "validate_channel_id": self.context.get("validate_channel_id", True),
```

Comment:
Setting the default to `True` rather than adding the value on a specific API. The code was originally written to default to True in `channel.py`.